src/SQLite3-Core-Tests/SQLiteConnectionTest.class.st

```diff
@@ -442,7 +473,7 @@ SQLiteConnectionTest >> testExecuteWithInsertBooleanSQLNullThenSelect [
 	self assert: res equals: 0.
 
 	rs := db execute: 'insert into x values (NULL, NULL)'.
-	self assert: rs next isNil.
+	self assert: rs next equals: nil.
```

Comment:
Why use the #assert:equals: here when using the #assert: with #isNil checking for a boolean is fine too?

Reply:
No good reason other than I had changed it to return something else for a bit and had not put it back exactly the same.  I will revert that.

Reply:
Reverted